src/chigame/forums/forum_conversation/models.py

```diff
@@ -0,0 +1,32 @@
+from django.conf import settings
+from django.db import models
+from machina.apps.forum_conversation.abstract_models import AbstractPost
+
+from chigame.users.models import User
+
+
+class Post(AbstractPost):
+    # Each post may be liked or disliked
+    ratings = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        through="Vote",
+        related_name="ratings",
+        editable=False,
+        blank=True,
+    )
+
+
+class Vote(models.Model):
+    class Rating(models.IntegerChoices):
+        LIKE = 1, "Like"
+        DISLIKE = -1, "Dislike"
+
+    poster = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
+    post = models.ForeignKey(Post, on_delete=models.CASCADE, null=True)
+    rating = models.IntegerField(choices=Rating.choices)
+
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["poster", "post"], name="unique_rating")]
+
+
+from machina.apps.forum_conversation.models import *  # noqa
```

Comment:
I added documentation for clarification.

TLDR: 'noqa' turns of the linter for the line since the import is at the bottom. It has to be at the bottom per django-machina instructions on overriding models.

Reply:
Ok, that makes sense and I appreciate the clarifying comment with link to the relevant documentation.